website/source/docs/providers/cloudstack/r/network.html.markdown

```diff
@@ -34,9 +34,23 @@ The following arguments are supported:
 * `cidr` - (Required) The CIDR block for the network. Changing this forces a new
     resource to be created.
 
+* `startip` - (Optional) Start of the IP block that will be available on the 
+    network. It will be autocalculated to the second available IP in the range
```

Comment:
It's more inline with the other docs to write these `It will be autocalculated to the second...` as `Defaults to second...`